map/apply-scheduled-updates-to-preplanned-map-area/src/main/java/com/esri/samples/apply_scheduled_updates_to_preplanned_map_area/ApplyScheduledUpdatesToPreplannedMapAreaSample.java

```diff
@@ -0,0 +1,234 @@
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapSyncParameters;
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapSyncResult;
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapSyncTask;
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapUpdatesInfo;
+import com.esri.arcgisruntime.tasks.offlinemap.OfflineUpdateAvailability;
+import com.esri.arcgisruntime.tasks.offlinemap.PreplannedScheduledUpdatesOption;
+
+import org.apache.commons.io.FileUtils;
+
+public class ApplyScheduledUpdatesToPreplannedMapAreaSample extends Application {
+
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) {
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Update Offline Map Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a map view
+      mapView = new MapView();
+
+      // create progress indicator
+      ProgressIndicator progressIndicator = new ProgressIndicator();
+      progressIndicator.setVisible(false);
+
+      // create a button to update the offline map
+      Button applyUpdatesButton = new Button("Apply Scheduled Updates");
+      applyUpdatesButton.setDisable(true);
+
+      // create labels to show update availability and size
+      Label updateAvailableLabel = new Label("Updates: ");
+      updateAvailableLabel.setTextFill(Color.WHITE);
+      Label updateSizeLabel = new Label("Update size: ");
+      updateSizeLabel.setTextFill(Color.WHITE);
+
+      // create a control panel for the UI elements
+      VBox controlsVBox = new VBox(6);
+      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.3)"), CornerRadii.EMPTY,
+              Insets.EMPTY)));
+      controlsVBox.setPadding(new Insets(10.0));
+      controlsVBox.setMaxSize(180, 110);
+      controlsVBox.getChildren().addAll(applyUpdatesButton, updateAvailableLabel, updateSizeLabel);
+
+      // create a temporary copy of the local offline map files, so that updating does not overwrite them permanently
+      Path tempMobileMapPackageDirectory = Files.createTempDirectory("canyonlands_offline_map");
+      tempMobileMapPackageDirectory.toFile().deleteOnExit();
+      Path sourceDirectory = Paths.get("./samples-data/canyonlands/");
+      FileUtils.copyDirectory(sourceDirectory.toFile(), tempMobileMapPackageDirectory.toFile());
+
+      // load the offline map as a mobile map package
+      MobileMapPackage mobileMapPackage = new MobileMapPackage(tempMobileMapPackageDirectory.toString());
+      mobileMapPackage.loadAsync();
+      mobileMapPackage.addDoneLoadingListener(() -> {
+        if (mobileMapPackage.getLoadStatus() == LoadStatus.LOADED && !mobileMapPackage.getMaps().isEmpty()) {
+
```

Comment:
you could add:
`ArcGISMap offlineMap = mobileMapPackage.getMaps().get(0));`

and use that on line 97 and 103 to avoid repetition